README.md

```diff
@@ -61,8 +78,13 @@ steps:
 
 ### `tinybird_datasource`
 
-**Optional**: This is the endpoint to which the workflow run data should be pushed to. (default: `ci_workflows`)
+**Required**: This is the endpoint to which the workflow run data should be pushed to. (default: `ci_workflows`)
```

Comment:
Optional is still correct here, because it defaults to `ci_workflows` if not set. Would need to change that in the `action.yml` however.

Reply:
Ah, perfect, I assumed that it was mandatory because it was defined as `required` in the `action.yml`. Changed it in both places now.